raster/src/main/scala/geotrellis/raster/reproject/ReprojectRasterExtent.scala

```diff
@@ -32,7 +32,7 @@ object ReprojectRasterExtent {
     val threshold = math.min(xThreshold, yThreshold)
 
     // Densify the extent to get a more accurate reprojection
-    val denseGeom = Polygon(Densifier.densify(extent.toPolygon.jtsGeom, threshold).asInstanceOf[com.vividsolutions.jts.geom.Polygon])
+    val denseGeom = Polygon(Densifier.densify(extent.toPolygon().jtsGeom, threshold).asInstanceOf[com.vividsolutions.jts.geom.Polygon])
```

Comment:
@lossyrob as it is (ofc it is more a convention question: http://docs.scala-lang.org/style/naming-conventions.html#parentheses): 

```scala
/** Return this extent as a polygon */
  def toPolygon(): Polygon =
    Polygon( Line((xmin, ymin), (xmin, ymax), (xmax, ymax), (xmax, ymin), (xmin, ymin)) )
```

and not 

```scala
def toPolygon = { ... }
```

I can roll it back btw :)